python/lsst/afw/geom/testUtils.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import absolute_import, division
+#
+# LSST Data Management System
+# Copyright 2015 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+"""Utilities that should be imported into the lsst.afw.coord namespace when lsst.afw.coord is used
+
+In the case of the assert functions, importing them makes them available in lsst.utils.tests.TestCase
+"""
+import numpy as np
+
+from .geomLib import Box2I, Box2D
+
+__all__ = ["BoxGrid"]
+
+class BoxGrid(object):
+    """!Divide a box into nx x ny smaller boxes
```

Comment:
Can you state how the boxes are divided?  Are there any overlaps? What happens to any remainder?